app/script/calling/CallingRepository.js

```diff
@@ -1136,7 +1133,17 @@ z.calling.CallingRepository = class CallingRepository {
       .then(callEntity => {
         const mediaType = this._getMediaTypeFromProperties(properties);
         const conversationName = callEntity.conversationEntity.display_name();
-        this.logger.info(`Incoming '${mediaType}' call in conversation '${conversationName}'`, callEntity);
+
+        this.callLogger.info(
+          {
+            data: {
+              default: [mediaType, conversationName],
+              obfuscated: [mediaType, this.callLogger.obfuscate(conversationId)],
+            },
+            message: `Incoming '{0}' call in conversation '{1}'`,
+          },
```

Comment:
I guess we should actually move all of these out into const objects for improved readability

Reply:
Not a good idea. Sometime there is 2 logging within the same function so we would need different const names and will therefore reduce readability instead of improving it.